extensions/blocks/dialogue/components/time-stamp-control.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	BaseControl,
+	__experimentalNumberControl as NumberControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+function checkValidValue( val, max ) {
```

Comment:
Nit: As this is doing more than checking the value should it be called something else? I'm not sure what, `validateValue`, `sanitizeValue`, `getBoundedValue` maybe?

Reply:
nide suggestion. `validateValue` sounds good to me. Updating...